src/common/TreeOps.h

```diff
@@ -32,37 +32,34 @@ class TermVisitor {
             unsigned int nextChild = 0;
         };
         // MB: Relies on an invariant that id of a child is lower than id of a parent.
```

Comment:
We should remove this comment, right?

Reply:
Good point, yes, we should remove it, I'll do it.